pkg/kubelet/pluginmanager/cache/actual_state_of_world.go

```diff
@@ -75,6 +75,18 @@ type actualStateOfWorld struct {
 
 var _ ActualStateOfWorld = &actualStateOfWorld{}
 
+// HandlerWithName holds information for handler and the Name of the plugin
+type HandlerWithName struct {
+	Handler PluginHandler
+	Name    string
+}
+
+// MapHandlerWithName contains the map from socket path to HandlerWithName
+type MapHandlerWithName struct {
```

Comment:
`SocketPluginHandlers`?. In general I would recommend naming things based on what they are, instead of what types they are comprised of.